src/server/db_slice.h

```diff
@@ -167,7 +167,8 @@ class DbSlice {
 
   // Either adds or removes (if at == 0) expiry. Returns true if a change was made.
   // Does not change expiry if at != 0 and expiry already exists.
-  bool UpdateExpire(DbIndex db_ind, PrimeIterator main_it, uint64_t at);
+  // Do not delete from expire_table when keep_expire is true.
```

Comment:
@Super-long  why not avoid calling this function instead of adding a parameter that tells it to do nothing?
also, I am not sure what does it even mean: to pass `at=0` and `keep_expire=true`.

Reply:
you should delete the comment as well @Super-long  :)